include/ibamr/IBHierarchyIntegrator.h

```diff
@@ -299,15 +333,37 @@ class IBHierarchyIntegrator : public IBTK::HierarchyIntegrator
      */
     SAMRAI::tbox::Pointer<INSHierarchyIntegrator> d_ins_hier_integrator;
 
+    /**
+     * The regrid CFL interval indicates the number of meshwidths a particle may
+     * move in any coordinate direction between invocations of the regridding
+     * process.
+     *
+     * @note Currently, when the CFL-based regrid interval is specified, it is
```

Comment:
@drwells a quick question --- if you request to regrid after each step, does this setting override that setting?

Reply:
Yes.